app/routes/task.py

```diff
@@ -0,0 +1,109 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from datetime import date
+import requests
+import os
+
+from app.models.task import Task
+from app.routes.routes_helper import get_valid_item_by_id
+
+
+SLACK_ENDPOINT = "https://slack.com/api/chat.postMessage"
+SLACK_OATH_TOKEN = os.environ.get('SLACK_BOT_USER_OATH_TOKEN')
+SLACK_HEADERS = {
```

Comment:
FANTASTIC use of constants, this is exactly the usecase of constants I was looking for!